docs/troubleshooting.adoc

```diff
@@ -0,0 +1,157 @@
+= Troubleshooting Landrush
```

Comment:
Yes, please see my comments above, the scope of the doc will be broadened to include ADB issues as we encounter them and yes they will be categorized as required.